### ISSUE TITLE
prefer-web-first-assertions replaces cases it shouldn't

### ISSUE THREAD

**Issue by thomas-phillips-nz** [playwright-community/eslint-plugin-playwright#205]
After enabling the `prefer-web-first-assertions` rule, it suggests an incorrect replacement in some of my code:

Before:
```
    const expectedCheckedStatus = true;
    expect(await page.locator('input').isChecked()).toBe(expectedCheckedStatus);
```

After:
```
    const expectedCheckedStatus = true;
    await expect(page.locator('input')).toBeChecked(expectedCheckedStatus);
```

The After code is invalid, `toBeChecked()` does not take a boolean argument

I am not sure if it should leave the code alone in this case, I understand you can add a `{checked: boolean}` option to `toBeChecked()`, I am just concerned that this rule has an auto-fix option which leaves you with invalid code.

**Comment by mskelton** [comment]
`toBeChecked` does accept arguments, but it should be replacing with 

```js
await expect(page.locator('input')).toBeChecked({ checked: expectedCheckedStatus });
```
